src/jsutils/__tests__/didYouMean-test.ts

```diff
@@ -1,7 +1,7 @@
 import { expect } from 'chai';
 import { describe, it } from 'mocha';
 
-import didYouMean from '../didYouMean';
+import { didYouMean } from '../didYouMean';
```

Comment:
Can you please remove all default exports in separate PR against `16.x.x`?

Reply:
Opened https://github.com/graphql/graphql-js/pull/2880 will remove this before I rebase